boards/shields/nrf7002_ek/nrf7002_ek_cpunet.overlay

```diff
@@ -0,0 +1,6 @@
+/* Copyright (c) 2022 Nordic Semiconductor ASA
```

Comment:
I think there is no need to provide this file, `nrf7002_ek_coex.overlay` could be used directly instead when needed.